target/linux/realtek/dts-5.10/rtl83xx_d-link_dgs-1210_poe.dtsi

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+&leds {
+	link_act {
+		label = "green:link_act";
+		gpios = <&gpio1 28 GPIO_ACTIVE_LOW>;
+	};
+
+	poe {
+		label = "green:poe";
+		gpios = <&gpio1 29 GPIO_ACTIVE_LOW>;
+	};
```

Comment:
It's not entirely clear to me how these work. Am I correct to understand from your commit message that each port has one of these LEDs? Or are these global status LEDs?

If these GPIOs hook up to a shift register, you can add an extra GPIO node for that, to expose all the LEDs. See https://elixir.bootlin.com/linux/v5.10.135/source/Documentation/devicetree/bindings/gpio/gpio-74x164.txt

Reply:
Answer is in comments below.